Geometry/HcalCommonData/interface/HcalDDDSimConstants.h

```diff
@@ -40,7 +40,7 @@ class HcalDDDSimConstants {
 					  const int zside) const;
   int                       getDepthEta16M(const int det) const;
   int                       getDepthEta29(const int phi, int zside, int i) const;
-  int                       getDepthEta29M(const int i) const;
+  int                       getDepthEta29M(const int i, const bool flag) const;
```

Comment:
@davidlange6 Done that - this is for special RBX's as in Plan One - so I name it planOne